lib/keys.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Creates keys for `private` properties on exposed objects to minimize interactions with other codebases.
+ * The key will be a Symbol if the host supports it; otherwise a prefixed string.
+ */
+var create = 
```

Comment:
I've changed to an `if/else` statement containing FDs for `create`. Not sure it reads better to be honest, but it's either that, ternary, or wrapping it in an IIFE (nasty IMO).